terminus-ui/card/src/card.component.spec.ts

```diff
@@ -1,95 +1,139 @@
+    });
+    test(`card is not flat`, () => {
+      component.flat = false;
+      fixture.detectChanges();
+      expect(cardComponent.flat).toEqual(false);
+      expect(card.classList).not.toContain('c-card--flat');
     });
   });
 
   describe(`supportsInteraction`, () => {
-    test(`should set and retrieve`, () => {
+    test(`card supports interaction`, () => {
       component.supportsInteraction = true;
-      expect(component.supportsInteraction).toEqual(true);
+      fixture.detectChanges();
+      expect(cardComponent.supportsInteraction).toEqual(true);
+      expect(card.classList).toContain('c-card--interaction');
+    });
+    test(`card does not support interaction`, () => {
+      component.supportsInteraction = false;
+      fixture.detectChanges();
+      expect(cardComponent.supportsInteraction).toEqual(false);
+      expect(card.classList).not.toContain('c-card--interaction');
     });
   });
 
+  describe(`centeredContent`, () => {
+    it(`should set and retrieve`, () => {
+      expect(cardComponent.centeredContent).toEqual(false);
+      cardComponent.centeredContent = true;
+      expect(cardComponent.centeredContent).toEqual(true);
+    });
+  });
 
   describe(`set aspectRatio()`, () => {
-
     it(`should convert the string aspect to a percentage`, () => {
-      component.aspectRatio = '16:9';
-      expect(component.aspectRatioPadding).toEqual('56.25%');
+      cardComponent.aspectRatio = '16:9';
+      expect(cardComponent.aspectRatioPadding).toEqual('56.25%');
 
-      component.aspectRatio = '4:3';
-      expect(component.aspectRatioPadding).toEqual('75.00%');
+      cardComponent.aspectRatio = '4:3';
+      expect(cardComponent.aspectRatioPadding).toEqual('75.00%');
     });
-
   });
 
-
   describe(`id`, () => {
 
     test(`should set a unique ID and be overwritable and default to UUID if no value is passed in`, () => {
-      expect(component.id).toBeTruthy();
-      expect(component2.id).toBeTruthy();
-      expect(component.id).not.toEqual(component2.id);
-
-      component.id = 'foo';
-      expect(component.id).toEqual('foo');
-
-      component.id = null as any;
-      expect(component.id).toEqual(component['_uid']);
-    });
-
-  });
+      expect(cardComponent.id).toBeTruthy();
+      expect(cardComponent2.id).toBeTruthy();
+      expect(cardComponent.id).not.toEqual(cardComponent2.id);
 
-  describe(`flat`, () => {
+      cardComponent.id = 'foo';
+      expect(cardComponent.id).toEqual('foo');
 
-    test(`should set flat flag correctly`, () => {
-      component.flat = true;
-      expect(component.flat).toEqual(true);
+      cardComponent.id = null as any;
+      expect(cardComponent.id).toEqual(cardComponent['_uid']);
     });
 
   });
 
-  describe(`isDisabled`, () => {
-
-    test(`should set isDisabled flag correctly`, () => {
-      component.isDisabled = true;
-      expect(component.isDisabled).toEqual(true);
-    });
-
-  });
-
-
   describe(`theme`, () => {
```

Comment:
We should test for the existence of theme classes in the DOM rather than the component property